bot/src/main/java/de/chojo/gamejam/data/GuildData.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 DevCord Team and Contributor
+ */
+
+package de.chojo.gamejam.data;
+
+import de.chojo.gamejam.data.wrapper.guild.Settings;
+import de.chojo.sqlutil.base.QueryFactoryHolder;
+import de.chojo.sqlutil.wrapper.QueryBuilderConfig;
+import net.dv8tion.jda.api.entities.Guild;
+
+import javax.sql.DataSource;
+import java.util.concurrent.CompletableFuture;
+
+public class GuildData extends QueryFactoryHolder {
```

Comment:
As far as I can see, this class is a DAO and should be named so, e.g. GuildDao

Reply:
I don't implement any DAO pattern. I don't use any abstraction and I also dont plan to do this.